### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
CC @vikivivi would you mind reviewing? TIA

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -13442,6 +13447,8 @@ ephememeris->ephemeris
 epidsodes->episodes
 epigramic->epigrammatic
 epilgoue->epilogue
+episdoe->episode
+episdoes->espisodes
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:13451]:
```suggestion
episdoes->episodes
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -16324,6 +16333,7 @@ functonality->functionality
 functoning->functioning
 functons->functions
 functtion->function
+functtioning->functioning
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:16336]:
```suggestion
functtional->functional
functtionalities->functionalities
functtioned->functioned
functtioning->functioning
functtions->functions
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -24994,6 +25005,7 @@ peroendicular->perpendicular
 perogative->prerogative
 peroid->period
 peroidic->periodic
+peroidically->periodically
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:25008]:
```suggestion
peroidical->periodical
peroidically->periodically
peroidicals->periodicals
peroidicity->periodicity
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -32064,6 +32079,7 @@ staps->steps, stops,
 staration->starvation
 stard->start
 stardard->standard
+stardards->standards
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:32082]:
```suggestion
stardardize->standardize
stardardized->standardized
stardardizes->standardizes
stardardizing->standardizing
stardards->standards
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -34012,6 +34029,7 @@ througout->throughout
 througt->through
 througth->through
 throuh->through
+throuhg->through
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:34032]:
```suggestion
throuhg->through
throuhgout->throughout
throuhgput->throughput
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -33131,6 +33147,7 @@ swithing->switching
 switiches->switches
 swown->shown
 swtich->switch
+swtiched->switched
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:33150]:
```suggestion
swtichable->switchable
swtichback->switchback
swtichbacks->switchbacks
swtichboard->switchboard
swtichboards->switchboards
swtiched->switched
swticher->switcher
swtichers->switchers
swtiches->switches
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -33131,6 +33147,7 @@ swithing->switching
...
 swtiching->switching
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:33151]:
```suggestion
swtiching->switching
swtichover->switchover
```

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
@vikivivi thx. added all your requested revisions. Please re-check. 
@DimitriPapadopoulos please review and approve (I'd like to propose @vikivivi as an official reviewer (if they'll agree of course))

**Comment by vikivivi** [comment]
LGTM
@luzpaz I can help but I preferred not been an official reviewer. Thank you for the offer.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**